js/Source/Request/Request.PollForUpdate.js

```diff
@@ -63,10 +63,10 @@ Request.PollForUpdate = new Class({
     var data = this.request.response.json;
     // if the server provides an 'updated_at' timestamp,
     // store as an attribute
-    if (data.status == 'update'){
+    if (typeof data !== 'undefined' && data.status == 'update'){
```

Comment:
...though I'll reiterate that it's overkill; if data is `false`, `null`, zero or an empty string you're still screwed and should throw an error, so the type check isn't merited.